docs/source/ko/tasks/sequence_classification.mdx

```diff
@@ -0,0 +1,391 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# 텍스트 분류[[text-classification]]
+
+[[open-in-colab]]
+
+<Youtube id="leNG9fN9FQU"/>
+
+텍스트 분류는 자연어 처리의 일종으로, 텍스트에 레이블 또는 클래스를 지정하는 작업입니다. 많은 대기업이 다양한 실용적인 응용 분야에서 텍스트 분류를 운영하고 있습니다. 가장 인기 있는 텍스트 분류 형태 중 하나는 감성 분석으로, 텍스트 시퀀스에 🙂 긍정, 🙁 부정 또는 😐 중립과 같은 레이블을 지정합니다.
+
+이 가이드에서 학습할 내용은:
+
+1. [IMDb](https://huggingface.co/datasets/imdb) 데이터셋에서 [DistilBERT](https://huggingface.co/distilbert-base-uncased)를 미세 튜닝하여 영화 리뷰가 긍정적인지 부정적인지 판단합니다.
+2. 추론을 위해 미세 튜닝 모델을 사용합니다.
+
+<Tip>
+이 튜토리얼에서 설명하는 작업은 다음 모델 아키텍처에 의해 지원됩니다:
```

Comment:
As a Korean speaker will rarely write colons(`:`) after finishing sentences, may you please edit preliminary sentences that explain code blocks to end with a full stop(`.`)?

Here are the corresponding line numbers:
- [ ] L27
- [ ] L38
- [ ] L44
- [ ] L54
- [ ] L61
- [ ] L72
- [ ] L79
- [ ] L87
- [ ] L121
- [ ] L145
- [ ] L156
- [ ] L160
- [ ] L170
- [ ] L221
- [ ] L234
- [ ] L244
- [ ] L262
- [ ] L280
- [ ] L291
- [ ] L297
- [ ] L303
- [ ] L317
- [ ] L323
- [ ] L346
- [ ] L356
- [ ] L365
- [ ] L374
- [ ] L383